src/App.svelte

```diff
@@ -1,5 +1,6 @@
 <script>
-  import Card from "./Card.svelte";
+  import { dataset_dev } from "svelte/internal";
```

Comment:
can you explain what this import does, because I couldn't find usages of `dataset_dev` on this file?

Reply:
Vs code adds stuff whenever something by mistake is called I forgot to remove it my bad 😅

Reply:
np 👍